### DISCUSSION TITLE
How to access the http request querystring parameters on the trame1.19.1

### DISCUSSION THREAD

**Discussion by chenjiajun1991** [Kitware/trame#91] [Q&A]
How would I access the http request querystring parameters on the trame1.19.1 ?

**Comment by jourdain** [comment]
This was answered [here](https://github.com/Kitware/trame/discussions/83)

**Comment by jourdain** [comment]
Oups sorry you were asking for trame v1 not v2. [Trame v1 component is here](https://github.com/Kitware/trame/blob/v1.20.0/trame/html/__init__.py#L576).
